tests/components/holiday/test_calendar.py

```diff
@@ -0,0 +1,39 @@
+"""Tests for calendar platform of Holiday integration."""
+
+from datetime import datetime
+
+from freezegun.api import FrozenDateTimeFactory
+
+from homeassistant.components.holiday import calendar
+from homeassistant.core import HomeAssistant
+from homeassistant.util import dt as dt_util
+
+from tests.common import MockConfigEntry
+
+
+async def test_async_get_events(
```

Comment:
We should set up the integration properly so we can test the state also

Reply:
Done.